.github/workflows/cd.yml

```diff
@@ -20,10 +20,14 @@ jobs:
       - uses: actions/setup-java@v1
         with:
           java-version: 1.8
+      - name: Install heroku cli
+        run: |
+          curl --silent --show-error --fail -O https://cli-assets.heroku.com/install.sh >> install.sh
```

Comment:
```suggestion
          curl --silent --show-error --fail -O https://cli-assets.heroku.com/install.sh
```

Reply:
no, this doesn`t work I have to save it explicitly to a file

Reply:
You sure? I tried locally and it works. Where did you try it? `-O` means that it will save to the file with the same name instead of outputting to console.